acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
```

Comment:
Yes, it would hit the exception below. Do we intend to start testing against those platforms?